apipie-bindings.gemspec

```
@@ -22,17 +22,16 @@ EOF
   s.extra_rdoc_files = `git ls-files -- {doc,config}/* README*`.split("\n")
   s.require_paths    = ["lib"]
 
-  s.add_dependency 'json', '>= 1.2.1'
-  s.add_dependency 'rest-client', '>= 1.6.5', '< 3.0.0'      # lower versions don't allow setting infinite timeouts, higher versions have different api
-  s.add_dependency 'oauth'
+  s.add_dependency 'json', '>= 2.3.0'
+  # lower versions don't allow setting infinite timeouts, higher versions have different api
+  s.add_dependency 'rest-client', '>= 2.1.0', '< 3.0.0'
```

I'd keep it even simpler:
```suggestion
  s.add_dependency 'rest-client', '~> 2.0'
```